### PR TITLE
Build bottles for arm64 in CI

### PR DESCRIPTION
**Pull request by kylewillmon**
Use self-hosted runners to build bottles for arm64 targets.

### CODE REVIEW

**Comment by kylewillmon** [comment]
@louislang, @brent-phylum: The GitHub hosted runners have Homebrew preinstalled. Our self-hosted runners do not. If it's simple to add Homebrew to our self-hosted runners, that would simplify the process of building arm64 bottles (i.e., pre-built binary packages)

**Comment by louislang** [comment]
I'll install it to the macos runner 👍

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -8,7 +8,11 @@ jobs:
   test-bot:
     strategy:
       matrix:
-        os: [ubuntu-22.04, macos-12]
+        os:
+          - ubuntu-22.04
+          - macos-12
```
Comment (cd-work) [inline .github/workflows/tests.yml:13]: Did you explicitly not choose ubuntu/macos-latest here? I wouldn't want to constantly bump this when we just want to use the latest anyway?

Comment (kylewillmon) [reply .github/workflows/tests.yml:13]: This file was originally created by `brew tap-new` and used explicit versions. But I agree that latest makes more sense.

**Comment by louislang** [comment]
@kylewillmon `brew` is now installed on the github macos arm runner.

**Review by maxrake — APPROVED** [review]

**Pull request merged**